services/scripts/testy-snapshot.sh

```diff
@@ -485,6 +487,10 @@ validate_database() {
         
     aws ec2 create-tags --resources "$_snapshot_id" "$_volume_id" \
                         --tags Key=Validation,Value=failed
+    
+    # Rename failure files to their associated snapshot id.
+    sudo mv -v "${failure_dir}${failure_file}" "${failure_dir}/${_snapshot_id}.txt"
```

Comment:
You can do this because you have the `/` as part of `failure_file`, I'd prefer this:
```suggestion
    sudo mv -v "${failure_dir}/${failure_file}" "${failure_dir}/${_snapshot_id}.txt"
```